src/main/resources/aws-smtp-relay.service

```diff
@@ -0,0 +1,14 @@
+# /usr/lib/systemd/system/tor.service
```

Comment:
this is the result of a copy/paste we should update

Reply:
Fixed